sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -729,6 +730,16 @@ private Mono<CosmosDatabaseResponse> createDatabaseInternal(Database database, C
                 requestOptions);
     }
 
+    /**
+     * Reads the Cosmos database account.
+     *
+     * @param shouldUseCache a boolean flag to determine whether to use the CosmosAsyncClient's-internal cache for reading the database account (setting shouldUseFlag to true can return stale data).
+     * @return the {@link CosmosDatabaseAccountResponse} with the read database account.
+     */
+    public Mono<CosmosDatabaseAccountResponse> readDatabaseAccount(boolean shouldUseCache) {
```

Comment:
NIT: I would probably prefer the name to ensure false is the default behavior - so shouldSkipCache or somethig like that